scripts/webpack/webpack.prod.js

```diff
@@ -63,7 +63,7 @@ module.exports = (env = {}) =>
     },
 
     // enable persistent cache for faster builds
-    cache: {
+    cache: parseInt(env.noMinify, 10) === 1 ? false : {
```

Comment:
Probably want to revert before merge?

Reply:
before merge to main, but i want to pull this out into own PR to land separately on main